roles/install_klipper/tasks/main.yml

```diff
@@ -94,17 +94,32 @@
       - python-dev
   when: platform == 'recore'
 
-- name: Recore - install Klipper with install scripts      
+- name: Recore - install Klipper with install scripts
   shell: "export USER=debian; {{ klipper_home }}/scripts/install-recore-a3.sh"
-  args: 
+  args:
     chdir: "{{ klipper_home }}"
   become: true
   become_user: debian
   when: platform == 'recore'
 
-- name: Pi/Pi64 - install Klipper with install scripts      
+- name: Recore - Make out folder
```

Comment:
... make out folder 🤔🤣

You love your code? 😂

Reply:
You know it : )